committee-steering/governance/wg-governance.md

```diff
@@ -0,0 +1,103 @@
+building.  If code is developed as part of collaboration within the Working Group, that code will be housed in an
+appropriate repository as described in the [repositories document].  The merging of this code into the repository
+will be governed by the standard policies regarding submitting code to that repository (e.g. developed within one or
+more Subprojects owned by SIGs).
+
+Because a working group is an official part of the Kubernetes project it is subject to steering committee oversight
+over its formation and disbanding.
+
+## Goals of the process
+
+- An easy-to-navigate process for those wishing to establish and eventually disband a new Working Group
+- Simple guidance and differentiation on where a Working Group makes sense, and does not
+- Clear understanding that no authority is vested in a Working Group
+- Ensure all future Working Groups conform with this process
+
+## Non-goals of the process
+
+- Documenting other governance bodies such as sub-projects or SIGs
+- Changing the status of existing Working Groups/SIGs/Sub-projects
+
+## Is it a Working Group? Yes, if...
+- It does not own any code
+- It has a clear goal measured through a specific deliverable or deliverables
+- It is temporary in nature and will be disbanded after reaching its stated goal(s)
+
+## Creation Process Description
+Working Group formation is less tightly-controlled than SIG formation since they:
+
+- Do not own code
+- Have a clear entry and exit criteria
+- Do not have any organizational authority, only influence
+
+Therefore, Working Group formation begins with the organizers asking themselves some important questions that
+should eventually be reflected in a pull request on sigs.yaml:
+
+1. What is the exact problem this group is trying to solve?
+1. What is the artifact that this group will deliver, and to whom?
+1. How does the group know when the problem solving process is completed, and it is time for the Working Group to
+   dissolve?
+1. Who are all of the stakeholders involved in this problem this group is trying to solve (SIGs, steering committee,
+   other Working Groups)?
+1. What are the meeting mechanics (frequency, duration, roles)?
+1. Does the goal of the Working Group represent the needs of the project as a whole, or is it focused on the interests
+   of a narrow set of contributors or companies?
+1. Who will chair the group, and ensure it continues to meet these requirements?
+1. Is diversity well-represented in the Working Group?
+
+Once the above questions have been answered, the pull request against sigs.yaml can be created. Once the generator
+is run, this will in turn create the OWNERS_ALIASES file, readme files, and the main SIGs list.  The minimum
+requirements for that are:
+
+- name
+- directory
+- mission statement
+- chair information
+- meeting information
+- contact methods
+- any sig stakeholders
+- any subproject stakeholders
```

Comment:
subproject here refers to those from other SIGs. right?

Reply:
#2814